ovs-service/src/test/java/org/dcsa/ovs/controller/ServiceScheduleControllerTest.java

```diff
@@ -97,7 +98,7 @@ void testGetServiceSchedulerReturns400ForInvalidVesselIMONumberLength() throws E
         .andExpect(jsonPath("$.requestUri").value("/service-schedules"))
         .andExpect(jsonPath("$.errors[0].reason").value("invalidInput"))
         .andExpect(
-            jsonPath("$.errors[0].message").value(containsString("size must be between 0 and 7")));
+            jsonPath("$.errors[0].message").value(containsString("findAll.vesselIMONumber must be a valid Vessel IMO Number")));
```

Comment:
`findAll.X` does not seem like a good name for the entity/attribute.  Is there a trivial way to change it so we do not use the internal method name as prefix?